rdmd.d

```diff
@@ -694,10 +714,13 @@ private string[string] getDependencies(string rootModule, string workDir,
     immutable rootDir = dirName(rootModule);
 
     // Collect dependencies
-    auto depsGetter =
-        // "cd "~shellQuote(rootDir)~" && "
-        [ compiler ] ~ compilerFlags ~
-        ["-v", "-o-", rootModule, "-I"~rootDir];
+    auto depsGetter = [ compiler ] ~ compilerFlags ~ [
+        "-v",
+        "-c",
+        "-of"~buildPath(objDir, "rdmd.root"~objExt),
+        rootModule,
+        "-I"~rootDir
+    ];
 
     scope(failure)
     {
```

Comment:
While `run` can throw (e.g. writing the output file), it just returns an exit code if the spawned process fails.
In `rebuild` errors are handled by just checking for the exit code.